### PR TITLE
Use platform specific CoreRun in API tasks

### PR DESCRIPTION
**Pull request by ellismg**
This should address dotnet/corefx#8815

### CODE REVIEW

File: src/Microsoft.DotNet.Build.Tasks/PackageFiles/ApiCompat.targets [hunk 1]
```diff
@@ -44,7 +44,7 @@
     </ItemGroup>
 
     <PropertyGroup>
-      <_ApiCompatCmd>"$(ToolsDir)CoreRun.exe" "$(ToolsDir)ApiCompat.exe" "$(ReferenceAssembly)" </_ApiCompatCmd>
+      <_ApiCompatCmd>$(ToolHostCmd) "$(ToolsDir)ApiCompat.exe" "$(ReferenceAssembly)" </_ApiCompatCmd>
```
Comment (weshaggard) [inline src/Microsoft.DotNet.Build.Tasks/PackageFiles/ApiCompat.targets]: There is one more location below with this same problem. Can you please update it as well?

**Comment by weshaggard** [comment]
Thanks for fixing this @ellismg. 

LGTM (after you fix the other location as well).

File: src/Microsoft.DotNet.Build.Tasks/PackageFiles/ApiCompat.targets (hunk 1 above)
Comment (ellismg) [inline src/Microsoft.DotNet.Build.Tasks/PackageFiles/ApiCompat.targets]: Thanks, fixed.

**Pull request merged**